package.json

```diff
@@ -23,7 +24,9 @@
     "grunt-contrib-uglify": "5.2.2",
     "grunt-contrib-watch": "1.1.0",
     "grunt-sass": "3.1.0",
-    "load-grunt-tasks": "5.1.0"
+    "load-grunt-tasks": "5.1.0",
+    "postcss-cli": "^10.1.0",
+    "tailwindcss": "^3.3.1"
   },
   "scripts": {
     "setup": "npm install --include=dev && composer install && composer dump-autoload -o",
```

Comment:
We need to add a command on how to run tailwind here.

Reply:
i have added command to run tailwind in script . this what you want me to do or something else?

Reply:
```suggestion
    "setup": "npm install --include=dev && composer install && composer dump-autoload -o && npx tailwindcss -i ./assets/css/tailwind-input-style.css -o ./assets/css/tailwind-output-style.css",
```

Reply:
as per your suggestion i had done .